docs/source/control.rst

```diff
@@ -175,6 +175,13 @@ allocated region. For example::
     >>> block.read(13)
     b"Hello, world!"
 
+This file-like wrapper is subscriptable so that new smaller file-like views of
+memory may be constructed::
```

Comment:
This file-like wrapper is **sliceable** so that new smaller file-like views of **subsets** of memory may be constructed::